string qsr_type
string[] all_possible_relations
time timestamp_request_received
time timestamp_qsrs_processed
ObjectsQSRs[] data